src/components/Articles.jsx

```diff
@@ -0,0 +1,32 @@
+import { useState, useEffect } from "react";
+import axios from "axios";
+import ArticleCard from "./ArticleCard.jsx";
+
+function ArticlesList() {
+  const [articles, setArticles] = useState([]);
+  useEffect(() => {
+    axios
```

Comment:
please extract this axios api call into a function within an api.js file.  Use axios create to create a base URL